README.md

```diff
@@ -1,91 +1,37 @@
-# Starter Kit
+## Tender Management System
+
+### Prerequisites
+  Ensure the following are installed on your system before proceeding:
+
+  - Docker Engine
+  - Docker Compose
+  - Node.js and npm (Node Package Manager)
+
+
+### ⏬ Installation and setup
+
+  To run this website locally, follow the following instructions:
+
+1. Clone the repo
+   ```sh
```

Comment:
nitpick. Add empty line before this line and remove the indentation. Do the same for all code blocks below